modules/registry-integrated-openshift-registry.adoc

```diff
@@ -1,22 +1,29 @@
 // Module included in the following assemblies:
 //
-// * assembly/registry
+//* assembly/registry
+//* registry/registry-options
+//* registry/architecture-component-imageregistry
 
 [id="registry-integrated-openshift-registry-{context}"]
 = Integrated {product-title} registry
 
-{product-title} provides an integrated container image registry called
-_{product-title} Container Registry_ (OCR) that adds the ability to automatically
-provision new image repositories on demand. This provides users with a built-in
-location for their application builds to push the resulting images.
+{product-title} provides a built in container image registry which runs as a
+standard workload on the cluster. The registry is configured and managed by an
+infrastructure operator. It provides an out of the box solution for users to
```

Comment:
The sentence starting with `It provides an out of the box...` should probably be broken apart into two sentences. What about:

  It provides an out of the box solution for users to manage the images that run their workloads, and runs on top of the existing cluster infrastructure. This registry can be scaled up or down like any other cluster workload and does not require specific infrastructure provisioning.